tests/on_wlm/test_containers_wlm.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from shutil import which
+
+from smartsim import Experiment, status
+from smartsim.entity import Ensemble
+from smartsim.settings.containers import Singularity
+
+"""Test SmartRedis container integration on a supercomputer with a WLM."""
+
+# Check if singularity is available as command line tool
+singularity_exists = which('singularity') is not None
+
+@pytest.mark.skipif(not singularity_exists, reason="Test needs singularity to run")
+def test_singularity_wlm_smartredis(fileutils, wlmutils):
+    """Run two processes, each process puts a tensor on
+    the DB, then accesses the other process's tensor.
+    Finally, the tensor is used to run a model.
+
+    Note: This is a containerized port of test_smartredis.py for WLM system
+    """
+
+    launcher = wlmutils.get_test_launcher()
+    if launcher not in ["pbs", "slurm"]:
+        pytest.skip(
+            "Test only runs on systems with PBS or Slurm as WLM"
+        )
+
+    test_dir = fileutils.make_test_dir()
+    exp = Experiment(
+        "smartredis_ensemble_exchange", exp_path=test_dir, launcher="slurm"
+    )
+
+    # create and start a database
+    orc = exp.create_database()
+    exp.generate(orc)
+    exp.start(orc, block=False)
+
+    container = Singularity('docker://benalbrecht10/smartsim-testing:latest')
```

Comment:
Could you make this a module file so that it's completely obvious to us whenever we get around to migrating the path to a craylabs one?

Reply:
I moved to a global variable per test. Also, we've updated to use a repo that @al-rigazzi has set up now, so there's no need to migrate this anymore (unless we eventually do create a CrayLabs dockerhub org).